base/array.jl

```diff
@@ -2325,7 +2325,7 @@ findall(testf::Function, A) = collect(first(p) for p in pairs(A) if testf(last(p
 
 # Broadcasting is much faster for small testf, and computing
 # integer indices from logical index using findall has a negligible cost
-findall(testf::Function, A::AbstractArray) = findall(testf.(A))
+findall(testf::Function, A::AbstractArray) = findall(map(testf, A))
```

Comment:
This would increase the allocation size as map return a `Array{Bool}` by default while `broadcast` return a `BitArray`.

Reply:
Okay, didn't know that. Thanks for noticing!